p2p/proto/consensus.proto

```diff
@@ -1,42 +1,115 @@
 syntax = "proto3";
 
-import "p2p/proto/common.proto";
-import "p2p/proto/transaction.proto";
-import "p2p/proto/state.proto";
+import "common.proto";
+import "transaction.proto";
+import "block.proto";
 import "google/protobuf/timestamp.proto";
 
 
 // WIP - will change
 
-message Proposal {
-    uint64                    block_number      = 2;
-    uint32                    round             = 3;
-    uint32                    pol               = 4;  // proof of lock
-    Hash                      block_header_hash = 5;
-    google.protobuf.Timestamp timestamp         = 6;
-    ConsensusSignature        signature         = 7;
+
+message BlockNumber {
```

Comment:
What do you think of BlockLocation?

The point of this field is to tell you where this block belongs in the chain. I also don't want anything like BlockId since these fields tell you nothing about the content of the block.